include/lsst/afw/image/VisitInfo.h

```diff
@@ -160,9 +162,10 @@ class VisitInfo : public table::io::PersistableFacade<VisitInfo>, public table::
     /**
     * Get rotation angle at boresight at middle of exposure
     *
-    * Rotation angle is angle of coordinate system specified by boresightRotType with respect to detector.
-    * For example if boresightRotType is SKY then at a rotation angle of 0 North is along detector Y axis
-    * (never flipped) and East is along the detector X axis (flipped if necessary)
+    * Rotation angle is position angle of focal plane +Y with respect to the coordinate system
+    * specified by rotType.  For example, if rotType is SKY, then at a rotation angle of 0,
+    * North is along focal plane +Y and East is along +X or -X (flipped to match handedness if
+    * necessary).  At rotation angle of 90, +Y will be aligned with E.
```

Comment:
The above is nice and clean, but wrong for `MOUNT`. Please try again. I suppose you could back off to something like "The meaning of the angle is defined by `rotType`, which see.", but if you can find a compact way to say more without misleading for `MOUNT` that would be nice.